spyder/plugins/completion/providers/languageserver/provider.py

```diff
@@ -69,6 +70,9 @@ class LanguageServerProvider(SpyderCompletionProvider):
         ('flake8/extendSelect', ''),
         ('flake8/extendIgnore', 'E,W,C90'),
         ('flake8/max_line_length', 79),
+        ('ruff/exclude', ''),
+        ('ruff/extendSelect', ''),
+        ('ruff/extendIgnore', 'D,DOC,E,W,C90'),
```

Comment:
```suggestion
        ('ruff/extendIgnore', 'E'),
```

To go with my suggestion above.